docs/UserGuide.md

```diff
@@ -180,7 +180,8 @@ IDs for insurance plans:
 * `INSURANCE_ID` match the associated insurance plan. For a list of mappings between insurance plans and 
 IDs, refer above.
 * `CLAIM_ID` is generated by your insurance company and should be given to you before adding the claim. However, it does not 
-affect the functionality here except for deleting this claim later, where it would be required.
+affect the functionality here except for deleting this claim later, where it would be required. Note that the claim ID is always
+capitalised in the system so `b1234` is the considered the same as `B1234`.
```

Comment:
Should we specify the CLAIM_ID format here to be clearer? e.g. Claim ID will always be in the format letter + 4 digits, this will be validated by the system and all claims in the wrong format will be rejected.

Reply:
Yeah can, I'll make the change later!